merge_condensed_sdrfs.py

```diff
@@ -75,6 +94,15 @@ def cluster_samples(cond: pd, main_covariate: str, covariate_type: str,
     :param batch_characteristic:
     :param batch_type:
     :return:
+    >>> accessions = ['E-GEOD-55866','E-GEOD-55482', 'E-CURD-31', 'E-GEOD-53197']
+    >>> input_path = get_test_fname("condensed-merge")
+    >>> cond_df = merged_condensed(accessions, input_path, batch_type='characteristic', batch_characteristic='study', new_accession="E-CURD-X") #doctest: +ELLIPSIS
+    Parsing ...
+    >>> conn_components = cluster_samples(cond=cond_df, main_covariate="organism part", covariate_type="characteristic", batch_characteristic="study", batch_type="characteristic", covariate_skip_values=[]) #doctest: +ELLIPSIS
```

Comment:
Specifying "organism part" is critical here. The default from the command line argument "organism_part" doesn't work.

Reply:
Changed the default now.